wordhelper/crossword_clue.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import urllib.parse
+from bs4 import BeautifulSoup
```

Comment:
This means we'll have to install the `bs4` module before using it. In that case you should install `bs4` as a dependency of our `wordhelper` module by running
```
>pipenv install bs4
```
so that `Pipfile` and `Pipfile.lock` will be updated accordingly.
Also, seeing that there are no changes in `Pipfile`, it probably means you installed `bs4` using `pip install`, which installed that module globally on your machine, which is something you don't want. So you might want to uninstall that globally first as well.
You can find more info about `pipenv`, including how to install *it* at https://github.com/pypa/pipenv.

Reply:
Done